.drone/drone.yml

```diff
@@ -243,6 +243,7 @@ steps:
       - go install github.com/tcnksm/ghr@latest
       - go install github.com/mitchellh/gox@latest
       - export PATH="$(go env GOPATH)/bin:$PATH"
+      - export RELEASE_DOC_TAG=`echo ${DRONE_TAG} | awk -F "\." '{print $1"."$2}'`
```

Comment:
This should probably be moved to the release bash script so it still works directly from make